secantTests/SnapshotTests/PrivateDataConsentSnapshotTests/PrivateDataConsentSnapshotTests.swift

```diff
@@ -15,7 +15,7 @@ class PrivateDataConsentSnapshotTests: XCTestCase {
         let store = Store(
             initialState: .initial
         ) {
-            PrivateDataConsentReducer(networkType: .mainnet)
```

Comment:
Does this test harness correctly initialize the dependency injector to inject mainnet? The vast majority of tests were using testnet, so I presume that is what the default setup for the test dependency injector uses.

Reply:
No, all tests are mostly network independent and while dependent it's always testnet. This injection here is just because the view needed the network from reducer. Newly the reducer can access it directly from TCA dependency.